plugin/src/main/java/io/jenkins/plugins/analysis/warnings/axivion/AxivionSuite.java

```diff
@@ -64,8 +65,13 @@ public final class AxivionSuite extends Tool {
     private String projectUrl = StringUtils.EMPTY;
     private String credentialsId = StringUtils.EMPTY;
     private String basedir = "$";
+
+    // field was added in 9.1.0
     private String namedFilter = StringUtils.EMPTY;
 
+    // field was added in 9.23.0
+    private boolean ignoreSuppressedOrJustified = true;
```

Comment:
Old serializations will all get a `false`. If this is not intended, then you need to change the field so that it returns the negation, e.g. something like `handleSuppressedAndJustified`

Reply:
`false` for old serializations is okay so no unexpected change will appear without changing the config